### DISCUSSION TITLE
How can I create an desktop- or application menu shortcut?

### DISCUSSION THREAD

**Discussion by ghost** [PhoenicisOrg/phoenicis#2325] [Q&A]
I wanna create shortcuts for the wine programs.

**Comment by plata** [comment]
Should happen automatically. Can you check `~/.local/share/applications`?

**Reply by ghost** [reply]
I have Krunner shortcuts for the programs,but I don't have desktop shortcuts,how can I create them?
And how can I delete the Krunner shortcuts?

**Reply by plata** [reply]
This is DE specific. Putting the `.desktop` files to `~/.local/share/applications` is correct. If you want KDE to show this on the desktop, you should talk to the KDE devs.

**Reply by ghost** [reply]
What is mean is,how can I create the desktop- and application menu in phoenicis and how can I delete them?
So I mean by right clicking and selecting "create desktop shortcut" etc.
Cause that isn't de specific if you can do it inside of phoenicis.

**Reply by plata** [reply]
This correlates to the shortcuts in your library.

**Reply by ghost** [reply]
Sorry,my english is shit,so I had to translate "correlates" and if that means "has to do with" here,then i don't get what you mean.
Does that what I wanna do work or not?

**Reply by plata** [reply]
Creating/deleting a shortcut will create/delete the `.desktop` file. Whether KDE/Gnome/... shows this on the desktop or only in the menu is not in our hands.

**Reply by ghost** [reply]
But Phoenicis has to select where to create the .desktop file.
So what I mean is,is there a way to create a .desktop file in `/home/toadfield/.local/share/applications/` (for application menu shortcuts) or `/home/toadfield/Desktop/` (for desktop shortcuts) in phoenicis?
Cause that has nothing to do with the de where the .desktop files will be created by phoenicis.

**Reply by plata** [reply]
As far as I'm aware, `~/Desktop` is DE specific and maybe even dependent on the language. We cannot use that.

**Reply by ghost** [reply]
But then,can't phoenicis "see" if what de the system uses?
Cause in programs like Lutris you can create desktop shortcuts,so it shouldn't be impossible to implement that afaik.
But for application menu shortcut it works,so is there a button to delete/create shortcuts for the application menu?

**Reply by plata** [reply]
> Cause in programs like Lutris you can create desktop shortcuts,so it shouldn't be impossible to implement that afaik.

Would have to check how they do it.

> But for application menu shortcut it works,so is there a button to delete/create shortcuts for the application menu?

Yes. It's create/delete shortcut in your library.

**Reply by ghost** [reply]
> Yes. It's create/delete shortcut in your library.

How can I use that?
If I click right click,then I don't see that.

**Reply by plata** [reply]
Advanced GUI, then you get a button in the left sidebar.
